### ISSUE TITLE
(Randomizer suggestion) Section to force techniques in a seed.

### ISSUE THREAD

**Issue by elreymu** [theonlydude/RandomMetroidSolver#14]
It would be cool to have a section in the advanced tab that can let you choose which techniques that the seed will require you to do (at least once)
For the purpose of being able to make seeds that for example always require a Lavadive in it, or always involve suitless maridia, or always involve doing a mid air springball jump, infinite bomb jumping, microwaving draygon, etc, at least once somewhere in the seed.
 
I came across this idea because a lot of times i find myself making a lot of seeds and checking if it involves techniques i enjoy doing (and some are rare to come by even with the presets), pretty sure i am not the only one, also i am curious how many techniques can be forced into a seed.
(I am aware that selecting too many conditions might cause the randomizer to abort or make a not realistic seed)

**Comment by theonlydude** [comment]
Hello,
We can't do that, we have to keep things random.
What we were thinking instead was to favor locations which required the most/least number of techniques (like we do with items which open the most/least number of new locations).
But we have to check that it won't be too slow first.

**Comment by elreymu** [comment]
Would it be possible to make it so it still keeps things random?

(Basically maximizing the number of required techniques in a seed (within the preset's known techniques) while keeping it random)

**Comment by theonlydude** [comment]
Yes, that's exactly what I meant with "favor locations which required the most/least number of techniques", but we have to check the performance of it first.
